packages/design/src/components/select-field/_variables.scss

```
@@ -0,0 +1,18 @@
+// TEXT
+$selectField-color-text-default: var(--fkds-color-text-primary);
```

@dvitamin Ska vi köra komponentens namn i camelCase eller ska vi köra kebab-case på allt? Jag tänker sistnämnda då det ser så blandat ut annars. Och i detta fallet kanske kanske tom att vi bara skriver ihop komponentens namn i lowercase?

Mitt förslag:

```suggestion
$selectfield-color-text-default: var(--fkds-color-text-primary);
```

Alternativt:

```suggestion
$select-field-color-text-default: var(--fkds-color-text-primary);
```

---

Håller med om kebab-case, kör på ditt förslag.

---

Fixat